services/gemma-local/audio_processor.py

```
@@ -0,0 +1,433 @@
+        
+        Args:
+            audio_base64: Base64-encoded audio data
+            prompt: Optional context prompt
+            
+        Returns:
+            Tuple of (audio_array, combined_prompt)
+        """
+        try:
+            # Validate base64 format
+            if not audio_base64 or len(audio_base64) == 0:
+                raise ValueError("Empty audio_base64 data provided")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)